webview/src/experiments/components/table/styles.module.scss

```diff
@@ -282,6 +290,12 @@ $workspace-row-edge-margin: $edge-padding - $cell-padding;
     }
   }
 
+  .table.withExpColumnShadow {
+    .tr:not(.rowSelected) > *:first-child:after {
+      box-shadow: 3px 0px 3px var(--vscode-widget-shadow);
```

Comment:
Should we add some `transition` to the `box-shadow` or is it fine as is? 

Also, currently, the header and experiments column are using two different box-shadow designs. Should I adjust the header box-shadow so they are using the same kind?